File: web-app/src/app/services/types/StudentSkill.ts
import { Skill } from "./Skill";
import { Tblstudent } from "./Tblstudent";

export type StudentSkill = {
  Id: number;
  StudentId?: number;
  SkillId?: number;
  Skill?: Skill;
  Student?: Tblstudent;
};

File: Placements.WebApi/Placement/Models/StudentSkill.cs
using System;
using System.Collections.Generic;

namespace Placements.WebApi.Placement.Models;

public partial class StudentSkill
{
    public long Id { get; set; }

    public long? StudentId { get; set; }

    public long? SkillId { get; set; }

    public virtual Skill? Skill { get; set; }

    public virtual Tblstudent? Student { get; set; }
}
